Applications/DataExplorer/DataView/GEOModels.cpp

```diff
@@ -74,114 +81,87 @@ void GEOModels::updateGeometry(const std::string &geo_name)
 void GEOModels::removeGeometry(std::string geo_name, GeoLib::GEOTYPE type)
 {
 	if (type == GeoLib::GEOTYPE::INVALID || type == GeoLib::GEOTYPE::SURFACE)
-		this->removeSurfaceVec(geo_name);
+		_geo_objects->removeSurfaceVec(geo_name);
 	if (type == GeoLib::GEOTYPE::INVALID || type == GeoLib::GEOTYPE::POLYLINE)
-		this->removePolylineVec(geo_name);
+		_geo_objects->removePolylineVec(geo_name);
 	if (type == GeoLib::GEOTYPE::INVALID || type == GeoLib::GEOTYPE::POINT)
-		this->removePointVec(geo_name);
+		_geo_objects->removePointVec(geo_name);
 }
 
-void GEOModels::addPointVec( std::vector<GeoLib::Point*>* points,
-                             std::string &name,
-                             std::map<std::string, std::size_t>* name_pnt_id_map,
-                             double eps)
+void GEOModels::addPointVec(std::string const& name)
 {
-	GEOObjects::addPointVec(points, name, name_pnt_id_map, eps);
-	_geoModel->addPointList(QString::fromStdString(name), *this->getPointVecObj(name));
+	_geoModel->addPointList(QString::fromStdString(name),
+	                        *_geo_objects->getPointVecObj(name));
 	emit geoDataAdded(_geoModel, name, GeoLib::GEOTYPE::POINT);
 }
 
-bool GEOModels::removePointVec( const std::string &name )
+void GEOModels::removePointVec(std::string const& name)
```

Comment:
- The result of any of the callbacks is not evaluated in GEOObjects.
 - There is no need to mimic the GEOObjects interface anymore.
 - The gui gives no feedback anyway; GeoTreeModel::removeGeoList has return type `void`.